app/assets/stylesheets/local/custom.scss

```diff
@@ -143,3 +143,8 @@ $app-blue-button-colour: govuk-colour("blue");
     }
   }
 }
+
+// To conditionally hide components to support JS/Non JS evidence upload functionality
+.app-evidence-upload-hidden {
+  display: none !important;
+}
```

Comment:
Ooooo `!important` is quite drastic - is that absolutely necessary?

Reply:
Actually no 😄 have removed